src/server/scripts/Kalimdor/HallsOfOrigination/boss_anraphet.cpp

```diff
@@ -327,24 +337,37 @@ class npc_brann_bronzebeard_anraphet : public CreatureScript
     public:
         npc_brann_bronzebeard_anraphet() : CreatureScript("npc_brann_bronzebeard_anraphet") { }
 
+        bool OnGossipHello(Player* player, Creature* creature) override
+        {
+            InstanceScript* instance = creature->GetInstanceScript();
+            if (!instance || instance->GetBossState(DATA_VAULT_OF_LIGHTS) == DONE)
+                return false;
+
+            player->PrepareGossipMenu(creature, GOSSIP_MENU_WE_ARE_READY, true);
+            player->SendPreparedGossip(creature);
+            return true;
+        }
+
         struct npc_brann_bronzebeard_anraphetAI : public CreatureAI
         {
-            npc_brann_bronzebeard_anraphetAI(Creature* creature) : CreatureAI(creature), _currentPoint(0), _instance(creature->GetInstanceScript()) { }
+            npc_brann_bronzebeard_anraphetAI(Creature* creature) : CreatureAI(creature), _currentPoint(0), _instance(creature->GetInstanceScript())
+            {
+                events.ScheduleEvent(EVENT_BRANN_IDLE_EMOTE, 45000);
```

Comment:
it should be on Reset(), and add std::chrono, like:

events.ScheduleEvent(EVENT_BRANN_IDLE_EMOTE, Seconds(45));